asapdiscovery-modeling/asapdiscovery/modeling/modeling.py

```diff
@@ -577,19 +577,24 @@ def split_openeye_mol(
         Molecule filter object that contains the filter criteria.
     prot_cutoff_len : int, default=10
         Minimum number of residues in a protein chain required in order to keep
+    keep_one_lig : bool, default=True
+        Only keep one copy of the ligand in lig_mol
 
     Returns
     -------
-    namedtuple
-        A namedtuple containing the protein, ligand, and water molecules.
+    dict
+        A dict containing the protein ("prot"), ligand ("lig"), water atoms ("wat"),
+        and all other atoms ("oth")
     """
     # These are the objects that will be returned at the end
     lig_mol = oechem.OEGraphMol()
     prot_mol = oechem.OEGraphMol()
     water_mol = oechem.OEGraphMol()
     oth_mol = oechem.OEGraphMol()
 
-    if type(molecule_filter) == str:
+    if molecule_filter is None:
+        molecule_filter = MoleculeFilter()
+    elif type(molecule_filter) == str:
```

Comment:
```suggestion
    elif type(molecule_filter) is str:
```
or  can use `isinstance` , happy with either, personally I think type(x) is ok for elementary types like str. 

https://www.flake8rules.com/rules/E721.html